pkg/terraformer/terraformer.go

```diff
@@ -510,3 +514,84 @@ func (t *terraformer) cleanupJob(ctx context.Context, jobPodList *corev1.PodList
 func (t *terraformer) computePodGenerateName(command string) string {
 	return fmt.Sprintf("%s.%s.tf-%s-", t.name, t.purpose, command)
 }
+
+//*******Terraform Raw State******
+
+// Marshal transform RawState to []byte representation. It encodes the raw state data
+func (trs *RawState) Marshal() ([]byte, error) {
```

Comment:
I guess you could split rawState related funcs to `pkg/terraformer/raw_state.go`.